Source/DafnyRuntime/DafnyRuntime.cs

```diff
@@ -18,7 +18,7 @@ public class DafnySourceAttribute : Attribute {
   }
 }
 
-namespace Dafny {
+namespace DafnyRuntime {
```

Comment:
I would be strongly in favor or having a sub-namespace rather than a concatenated namespace, so for example:
* `Dafny.Runtime`
* `Dafny.Run`
* `Dafny.VM`
* `Dafny.Library`
* `Dafny.Lib`
* `Dafny.System`
* `Dafny.Core`
* `Dafny.Include`

What do you think of that?